smooth/components/component.py

```diff
@@ -124,15 +124,19 @@ def update_var_costs(self, results, sim_params):
             self.results['art_costs'] = [0] * sim_params.n_intervals
 
         # Update the costs for this time step [EUR].
+        # Costs are associated with absolute values (e.g. [Wh] org [kg]), flows are given in
```

Comment:
typo: or instead of org
Consider writing these two lines at the top -> write a docstring for this function. This way, the same typo isn't duplicated.

Reply:
I added a docstring